fastdds_python/test/api/test_waitset.py

```diff
@@ -109,11 +125,21 @@ def test_waitset(datareader, datawriter):
         elif ('GuardCondition' == str(c)):
             try:
                 attached_guard_cond = c.to_status_condition()
+                attached_guard_cond = c.to_read_condition()
```

Comment:
This conversion will never be executed. It should be in another try-catch block

Reply:
This issue has been fixed in https://github.com/eProsima/Fast-DDS-python/pull/40